cluster.go

```diff
@@ -47,32 +47,34 @@ import (
 
 type mongoCluster struct {
 	sync.RWMutex
-	serverSynced  sync.Cond
-	userSeeds     []string
```

Comment:
So things are still getting reformatted. Perhaps the code in master isn't gofmt:ed.